src/Template/Element/json_meta_config.ctp

```diff
@@ -1,9 +1,9 @@
 <?php
     $csrfToken = null;
-    if (!empty($this->request->params['_csrfToken'])) {
-        $csrfToken = json_encode($this->request->params['_csrfToken']);
+    if (!empty($this->request->getParam('_csrfToken'))) {
+        $csrfToken = json_encode($this->request->getParam('_csrfToken'));
     } elseif (!empty($this->request->data['_csrfToken'])) {
-        $csrfToken = json_encode($this->request->data['_csrfToken']);
+        $csrfToken = json_encode($this->request->getParam('_csrfToken'));
```

Comment:
typo?
it should be `$csrfToken = json_encode($this->request->getData('_csrfToken'));` or somethink like that...